packages/webapp/pages/feeds/new.tsx

```diff
@@ -180,10 +176,6 @@ const NewFeedPage = (): ReactElement => {
     });
   }, [logEvent]);
 
-  if (showPlusSubscription) {
-    return null;
-  }
```

Comment:
We can just return null always, to prevent UI blink while redirect happens (also on edit